src/areas/About/Privacy.jsx

```diff
@@ -22,10 +22,11 @@ function Privacy() {
 
                 <Typography color="primary" variant="h5">Terms of Service</Typography>
                 <pre className="pre-info">
-                    WC2022 is just for fun, <i>should</i> be free, and has no attached warranty.<br/><br/>
-                    Again, this is in no way officially associated with FIFA or the FIFA World Cup Qatar 2022™.<br/><br/>
+                    WWC2023 is just for fun, <i>should</i> be free, and has no attached warranty.<br/><br/>
+                    Again, this is in no way officially associated with FIFA or the FIFA Women's World Cup 2023™.<br/><br/>
 
-                    Soccer ball icon made by <Link href="https://www.flaticon.com/authors/darius-dan" title="Darius Dan">Darius Dan</Link> from <Link href="https://www.flaticon.com/" title="Flaticon">www.flaticon.com</Link>
+                    {/*Unsure of whether the following commented line is the artist for WWC2023 icon*/}
```

Comment:
Yes, this is the updated trademark info for the new logo. We can add this back in.